src/App.tsx

```diff
@@ -1,61 +1,27 @@
 import './App.scss';
+import { useState, FC } from 'react';
+import todosFromServer from './api/todos';
```

Comment:
[REACT KNOWLEDGE] - Prepare data in one place (App component) and pass it to child components. Don't use `import` keyword in non-root components. In your case, you're importing `todosFromServer` directly in the App component which is fine, but make sure you don't do this in child components.